alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,88 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+The `PrometheusOperatorRejectedResources` alert triggers when Prometheus Operator
+rejects  `Alertmanager` `PodMonitor`, `ServiceMonitor`, or `PrometheusRules` objects
+because custom resources in an object are not configured correctly.
+
+## Impact
+
+The custom resources that trigger the alert are ignored by Prometheus Operator.
+As a consequence, they will not be part of the final configuration of the
+Prometheus, Alertmanager, or Thanos Ruler components managed by Prometheus Operator.
+
+## Diagnosis
+
+Find the type of custom resource and its namespace in the `description` field
+of the alert `PrometheusOperatorRejectedResources`.
+
+By CLI
+```bash
+### Gets alertmanager URL
+$ ALERT_MANAGER=$(oc get route alertmanager-main -n openshift-monitoring -o jsonpath='{@.spec.host}')
+
+### Gets fired alerts
+$ curl -k -H "Authorization: Bearer $(oc create token prometheus-k8s -n openshift-monitoring)"  https://$ALERT_MANAGER/api/v1/alerts
+```
+
+In Openshift web console:
+Browse to the tab Observe -> Alerting, then search the alert `PrometheusOperatorRejectedResources`
+in firing alerts list.
+![console alert list](PrometheusOperatorRejectedResources-alert-list.png)
+
+Click on the alert to see its details and scroll down to see its `Description` field.
+The namespace should be displayed as a link, followed by the resource type.
+
+![console alert detail](PrometheusOperatorRejectedResources-alert-detail.png)
+
+Then check recently modified custom resource of that type in that namespace.
+If the namespace is named `openshift-.*` except `openshift-user-workload-monitoring`,
+for example, `openshift-monitoring`, this may be a bug in Openshift,
+please file a support case.
```

Comment:
```suggestion
After you know the namespace and the resource type, you can look for a recently modified custom resource of that type in that namespace.
**Note**: If the namespace is named `openshift-.*`, (excluding`openshift-user-workload-monitoring`), this might be a bug in OpenShift. Please file a support case.
```